pkg/util/iptables/iptables.go

```diff
@@ -284,6 +284,42 @@ func (runner *runner) restoreInternal(args []string, data []byte, flush FlushFla
 	return nil
 }
 
+// ParseTableSaveChains parses a table's save and returns a map of Chain to chain in iptables-save format
```

Comment:
This comment and function name are not clear.  GetChainLines() maybe?  I'm all for extracting reusable logic, but this is on the hairy edge of not reusable enough,  That said, I don;t think you should parse further without real need.  I would just move this back to proxier and call it getChainLines()